code/modules/reagents/machinery/chem_dispenser.dm

```diff
@@ -339,6 +337,14 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 			to_chat(user, "You can't add \a [D] to the machine while the panel is open.")
 			return
 
+/obj/machinery/chem_dispenser/slime_act(primarytype, mob/user)
+	..()
+	if(primarytype == /mob/living/carbon/slime/black)
+		has_slime=1
+		dispensable_reagents.Add(DSYRUP)
+		to_chat(user, "You throw the slime into the dispenser's tank.")
```

Comment:
"You scan the slime with the dispenser's manipulators, unlocking a new reagent." or along those lines would be better and move obvious.

Reply:
eneocho moment